src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -545,6 +545,20 @@ def create_lightning_service(self, tank) -> client.V1Service:
         self.log.debug(f"Created lightning service {service_name} for tank {tank.index}")
         return service
 
+    def fetch_ip_address(self, tank) -> bool:
+        pod_ip = None
+        while not pod_ip:
+            pod_name = self.get_pod_name(tank.index, ServiceType.BITCOIN)
+            pod = self.get_pod(pod_name)
+            if pod is None or pod.status is None or getattr(pod.status, "pod_ip", None) is None:
+                print(f"Waiting for tank {tank.index} response or tank IP...")
+                time.sleep(3)
```

Comment:
in "k8s: remove ipv4 from k8s backend" (https://github.com/bitcoin-dev-project/warnet/pull/254/commits/1a3223d0accd8398954f36d402e17844b33be8db):

I don't have a good sense of how likely we are to hit this, but if we hit it on every single pod while spinning up a 500 node network, that would be `(3 sec * 500) / 60 sec = 25 mins`. maybe not a blocker for now, but wanted to mention it as i think this approach will need reworking for k8s.